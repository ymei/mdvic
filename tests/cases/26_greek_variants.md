Variants: $\varepsilon$, $\varphi$, $\vartheta$, $\varrho$, $\varsigma$.

