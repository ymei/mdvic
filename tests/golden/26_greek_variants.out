Variants: ϵ, ϕ, ϑ, ϱ, ς.

